scanner.py

```diff
@@ -1,12 +1,18 @@
 #!/usr/bin/env python
 import socket
 import subprocess
+import json
 import sys
 from datetime import datetime
 
 # Clear the screen
 subprocess.call('clear', shell=True)
 
+with open('config.json') as file:
+  data_json = json.load(file)
```

Comment:
@ishika1727 Ideally we should always have self-explanatory variables name. It makes the code one level up to read.

How about changing `data_json` to `config`?
Any thoughts on this?